buildkite/pipelines/gerrit-postsubmit.yml

```diff
@@ -2,31 +2,31 @@
 platforms:
```

Comment:
Was the reformatting intended?

Reply:
It has harder to convince VS Code to not format it, than to just do it ^^